### ISSUE TITLE
[SampleCard-ExpenseReport; Config-skype.json]: The arrow image have insufficient contrast ratio of 2.71:1 and should be atleast of 3:1

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7377]
https://microsoft.visualstudio.com/OS/_workitems/edit/37987199
